LICENSE

```diff
@@ -0,0 +1,671 @@
+                    GNU AFFERO GENERAL PUBLIC LICENSE
```

Comment:
Our current plan (always up for re-assessment if there is reason) is to use AGPL for XBlocks: https://open.edx.org/open-edx-licensing